11ty/definitions/mx.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Mx.
+slug: mx
+speech: noun
+defined: true
+excerpt: someone who is sexually attracted to people regardless of their gender, or to people of any gender.
```

Comment:
Was this copied from somewhere, doesn't look quite right?

Reply:
Ooh, good catch. thank you!